public/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -259,28 +276,26 @@ Light client mode downloads all block headers, block data, and verifies some ran
 
 **NB** Light sync does not yet work with proof-of-stake Ethereum - new versions of light sync should ship soon!
 
-[More on light clients](/developers/docs/nodes-and-clients/light-clients/)
+[More on light clients](/developers/docs/nodes-and-clients/light-clients/).
 
 ### Consensus layer sync modes {#consensus-layer-sync-modes}
 
 #### Optimistic sync {#optimistic-sync}
 
 Optimistic sync is a post-merge synchronization strategy designed to be opt-in and backwards compatible, allowing execution nodes to sync via established methods. The execution engine can _optimistically_ import beacon blocks without fully verifying them, find the latest head, and then start syncing the chain with the above methods. Then, after the execution client has caught up, it will inform the consensus client of the validity of the transactions in the Beacon Chain.
 
-[More on optimistic sync](https://github.com/ethereum/consensus-specs/blob/dev/sync/optimistic.md)
+[More on optimistic syncs](https://github.com/ethereum/consensus-specs/blob/dev/sync/optimistic.md).
 
 #### Checkpoint sync {#checkpoint-sync}
 
-Checkpoint sync, also known as weak subjectivity sync, creates a superior user experience for syncing Beacon Node. It's based on assumptions of [weak subjectivity](/developers/docs/consensus-mechanisms/pos/weak-subjectivity/) which enables syncing Beacon Chain from a recent weak subjectivity checkpoint instead of genesis. Checkpoint sync makes the initial sync time significantly faster with similar trust assumptions as syncing from [genesis](/glossary/#genesis-block).
+A checkpoint sync, also known as weak subjectivity sync, creates a superior user experience for syncing a Beacon Node. It's based on assumptions of [weak subjectivity](/developers/docs/consensus-mechanisms/pos/weak-subjectivity/) which enables syncing the Beacon Chain from a recent weak subjectivity checkpoint instead of genesis. Checkpoint syncs make the initial sync time significantly faster with similar trust assumptions as syncing from [genesis](/glossary/#genesis-block).
 
-In practice, this means your node connects to a remote service to download recent finalized states and continues verifying data from that point. Third party providing the data is trusted and should be picked carefully.
+In practice, this means your node connects to a remote service to download recent finalized states and continues verifying data from that point. The third party providing the data is trusted and should be picked carefully.
 
-More on [checkpoint sync](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice)
+More on [checkpoint syncs](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice).
```

Comment:
I'm not sure I'm a fan of changing to "syncs" here, but not a hill I'll die on

Reply:
don't have a rationale other than "it sounds better out loud"!